packages/dashboard/src/components/sidePanel/utils/index.ts

```diff
@@ -1,18 +1,34 @@
-import { useEffect, Dispatch, useState } from 'react';
+import { Dispatch, useEffect, useState } from 'react';
 // import { useDispatch } from 'react-redux';
-import { get, set, cloneDeep } from 'lodash';
+import { get } from 'lodash';
+import { useSelector } from 'react-redux';
+import { DashboardState } from '../../../store/state';
 
 export const useInput: <T>(path: string) => [T, Dispatch<T>] = (path) => {
-  const selectedWidget = {}; // TODO: get selected widget from state manager or context
-  const [value, updateValue] = useState(get(selectedWidget, path));
-  // const dispatch = useDispatch();
+  const [selectedWidget] = useSelector((state: DashboardState) => state.dashboardConfiguration.widgets);
+  const [value, updateState] = useState(get(selectedWidget, path));
```

Comment:
Is there a better name for value? Also, it would be easy to forget updateState is related to value due to their unrelated generic naming.